package.json

```diff
@@ -8,8 +8,8 @@
   "type": "module",
   "scripts": {
     "build": "aegir build",
-    "test": "aegir test",
-    "test:browser": "aegir test --target browser",
+    "clean": "aegir clean ; rm -rf node_modules/ ; rm -f package-lock.json yarn.lock",
```

Comment:
Curious as to why there's a need to remove the lock files?

Reply:
Dumb paranoia on my part. I'm fine with getting rid of that script - don't think anyone else is running it anyhow.